packages/core/src/App/Components/Layout/Header/wallets/account-info-wallets.tsx

```diff
@@ -76,10 +76,12 @@ const MobileInfoIcon = ({ currency, is_virtual }: Partial<TMobileInfoIcon>) => (
     </span>
 );
 
-const DesktopInfoIcons = observer(({ wallet_account }: TDesktopInfoIcons) => {
+const DesktopInfoIcons = observer(({ wallet_account, active_account }: TDesktopInfoIcons) => {
     const { ui } = useStore();
     const { is_dark_mode_on } = ui;
 
+    const show_badge = wallet_account?.is_malta_wallet || wallet_account?.is_virtual;
```

Comment:
@sergei-deriv add this logic to the parent of this component and make the badge an optional props so whenever you don't pass badge to it, it will be empty

Reply:
@mahdiyeh-deriv What do you think if I'll add this logic inside of the WalletBadge component?

Reply:
@mahdiyeh-deriv I have a suggestion, let's add show_badge as prop to WalletBadge component, in result it will be really dumb and logic to show/hide it will be in his parent? What do you think?

Reply:
@sergei-deriv yes, let's make the badge component a dummy one without any logic in that